qemu/tests/cfg/netdev_socket.cfg

```diff
@@ -0,0 +1,11 @@
+- netdev_socket:
+    type = netdev_socket
+    start_vm = no
+    vms = "vm1 vm2"
+    ip_vm1 = "192.168.10.1"
+    ip_vm2 = "192.168.10.2"
+    netdev_extra_params_vm1 = ",listen=127.0.0.1:12233"
+    netdev_extra_params_vm2 = ",connect=127.0.0.1:12233"
+    image_snapshot = yes
```

Comment:
@xiywang I would suggest adding "nettype=socket" here in cfg. This will simplify our test loop running, and we can be able to cover this test case with default nettype value.

Reply:
Great advice!
@pezhang @ybduan Updated. Please help to review again, thanks.